package.json

```diff
@@ -103,15 +104,10 @@
     "webpack-cli": "^4.9.1"
   },
   "resolutions": {
-    "ssri": ">=8.0.1",
-    "redis": ">=3.1.2",
-    "hosted-git-info": ">=3.0.8",
-    "lodash": ">=4.17.21",
-    "ws": ">=7.4.6",
-    "glob-parent": ">=5.1.2",
-    "set-value": ">=4.0.1",
-    "acorn": ">=5.7.4",
-    "ansi-regex": "<6.0.0",
-    "postcss": ">=8.2.13"
+    "redisscan/redis": "^3.1.2",
```

Comment:
`^` seems to be more "foolproof" than `>=`, because that ends up automatically updating the dependency to the latest version each time you run `yarn install` which can have hard to spot unintended consequences as it can happen while updating an unrelated package